docs/standard/components.md

```diff
@@ -14,37 +14,36 @@ ms.assetid: 2e38e9d9-8284-46ee-a15f-199adc4f26f4
 
-The .NET Standard is also a build target. You can currently target .NET Standard 1.0-2.0. If your code targets a version of the .NET Standard, it is guaranteed to run on any .NET runtime which implements that version.
+The .NET Standard is also a target framework. You can currently target .NET Standard 1.0-2.0. If your code targets a version of the .NET Standard, it is guaranteed to run on any .NET implementation which supports that version of .NET Standard.
 
 To learn more about the .NET Standard and how to target it, see the [.NET Standard](net-standard.md) topic.
 
-## .NET runtimes
+## .NET implementations
 
-There are 3 primary .NET runtimes which Microsoft actively develops and maintains: .NET Core, .NET Framework, and Mono for Xamarin.
+There are 3 primary .NET implementations which Microsoft actively develops and maintains: .NET Core, .NET Framework, and Mono for Xamarin.
 
 ### .NET Core
 
-.NET Core is a cross-platform runtime optimized for server workloads.  It implements the .NET Standard, which means that any code that targets the .NET Standard can run on .NET Core.  It is the runtime used by ASP.NET Core and the Universal Windows Platform (UWP).  It is modern, efficient, and designed to handle server and cloud workloads at scale.
+.NET Core is a cross-platform runtime optimized for server workloads.  It implements the .NET Standard, which means that any code that targets the .NET Standard can run on .NET Core.  It is the .NET implementation used by ASP.NET Core and the Universal Windows Platform (UWP).  It is modern, efficient, and designed to handle server and cloud workloads at scale.
 
 To learn more about .NET Core, see the [.NET Core Guide](../core/index.md).
 
 ### .NET Framework
 
-.NET Framework is the historical .NET runtime that has existed since 2002.  It is the same .NET Framework existing .NET developers have always used.  It implements the .NET Standard, which means that any code that targets the .NET Standard can run on the .NET Framework.  It contains additional Windows-specific APIs, such as APIs for Windows desktop development with Windows Forms and WPF.  .NET Framework is optimized for building Windows desktop applications.
+.NET Framework is the original .NET implementation that has existed since 2002.  It is the same .NET Framework that existing .NET developers have always used.  It implements the .NET Standard, which means that any code that targets the .NET Standard can run on the .NET Framework.  It contains additional Windows-specific APIs, such as APIs for Windows desktop development with Windows Forms and WPF. The .NET Framework is optimized for building Windows desktop applications.
```

Comment:
> The .NET Framework ...

It is :point_right: It's

For ...

> It implements the .NET Standard

... since that hasn't always been the case, I wonder if you want to qualify it with something like "Now, ..." or "Since \<year>, ..." to indicate it's a relatively recent feature.

More "it" subjects here. I recommend upgrading a few of these to concrete subjects.

> ... any code that targets the .NET Standard ...

"any" ... Again, I think there were (historical) examples of this not being true.

Two-spaces between sentences here.